core-java-modules/core-java-lang-3/src/test/java/com/baeldung/stringtoboolean/StringToBooleanUnitTest.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.stringtoboolean;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class StringToBooleanUnitTest
+{
```

Comment:
let's put this little guy inline with the class definition

Reply:
The opening bracket has been moved up.